layouts/_default/list.atom.xml

```diff
@@ -32,14 +32,211 @@
         {{- $copyright = $copyright | replace "&copy;" "©" -}}
         <rights>{{ $copyright | plainify }}</rights>
     {{- end }}
-    {{ with site.Author.name -}}
-        <author>
-            <name>{{ . }}</name>
-            {{ with site.Author.email }}
-                <email>{{ . }}</email>
-            {{ end -}}
-        </author>
-    {{- end }}
+    {{ $authors := "" }}
+    {{ if isset site "Author" }}
+        {{ $authors = site.Author }}
+    {{ else if isset site "Authors" }}
+        {{ $authors = site.Authors }}
+    {{ else if isset site "author" }}
+        {{ $authors = site.author }}
+    {{ else if isset site "authors" }}
+        {{ $authors = site.authors }}
+    {{ else if isset site.Params "Author" }}
```

Comment:
The rest of these 4 conditions can be replaced with something like:

```
{{ with (or (site.Param "author") (site.Param "authors")) }}
```

Notice the use of `Param` instead of `Params`. `site.Param "foo"` fetches any "foo" from site config's Params case-**insensitively**.